lib/compiler/src/beam_jump.erl

```diff
@@ -411,12 +411,18 @@ opt_useless_loads([{test,_,{f,L},_}=I|Is], L, St) ->
 opt_useless_loads(Is, _L, St) ->
     {Is,St}.
 
-opt_useless_block_loads([{set,[Dst],_,_}=I|Is], L, Index) ->
+opt_useless_block_loads([{set,[Dst],_,Op}=I|Is], L, Index) ->
```

Comment:
The compiler code has a particular style and we are particular that it should be relatively consistent.

When a variable is modified, the first instance should start out with a `0` suffix. So `Is` should be changed to `Is0`.